apigentools/utils.py

```diff
@@ -215,10 +222,17 @@ def write_full_spec(config, spec_dir, version, full_spec_file):
                 full_spec["paths"].update(loaded.get("paths", {}))
                 full_spec["tags"].extend(loaded.get("tags", []))
                 full_spec["components"]["schemas"].update(loaded.get("components", {}).get("schemas", {}))
+                full_spec["components"]["parameters"].update(loaded.get("components", {}).get("parameters", {}))
                 full_spec["components"]["securitySchemes"].update(loaded.get("components", {}).get("securitySchemes", {})),
+                full_spec["components"]["requestBodies"].update(loaded.get("components", {}).get("requestBodies", {})),
+                full_spec["components"]["responses"].update(loaded.get("components", {}).get("responses", {})),
+                full_spec["components"]["headers"].update(loaded.get("components", {}).get("headers", {})),
+                full_spec["components"]["examples"].update(loaded.get("components", {}).get("examples", {})),
+                full_spec["components"]["links"].update(loaded.get("components", {}).get("links", {})),
+                full_spec["components"]["callbacks"].update(loaded.get("components", {}).get("callbacks", {})),
```

Comment:
I think there's a more clever way to update this dict provided we know the name of the keys but I wanted to keep the diff mininal to ease review.

Reply:
You're right, that's something that we can definitely iterate on. Apigentools is still pretty fresh, so there's a lot of room for improvement in a lot places.